core-shaded/pom.xml

```diff
@@ -28,97 +28,197 @@
-                  <resources>
-                    <resource>META-INF/MANIFEST.MF</resource>
-                    <resource>META-INF-shaded/MANIFEST.MF</resource>
-                    <resource>META-INF/maven/com.datastax.oss/java-driver-core/pom.xml</resource>
-                    <resource>META-INF/maven/com.datastax.oss/java-driver-core/pom.properties</resource>
-                    <resource>META-INF/maven/io.netty/netty-handler/pom.properties</resource>
-                    <resource>META-INF/maven/io.netty/netty-handler/pom.xml</resource>
-                    <resource>META-INF/maven/io.netty/netty-buffer/pom.properties</resource>
-                    <resource>META-INF/maven/io.netty/netty-buffer/pom.xml</resource>
-                    <resource>META-INF/maven/io.netty/netty-common/pom.properties</resource>
-                    <resource>META-INF/maven/io.netty/netty-common/pom.xml</resource>
-                    <resource>META-INF/maven/io.netty/netty-transport/pom.properties</resource>
-                    <resource>META-INF/maven/io.netty/netty-transport/pom.xml</resource>
-                    <resource>META-INF/maven/io.netty/netty-resolver/pom.properties</resource>
-                    <resource>META-INF/maven/io.netty/netty-resolver/pom.xml</resource>
-                    <resource>META-INF/maven/io.netty/netty-codec/pom.properties</resource>
-                    <resource>META-INF/maven/io.netty/netty-codec/pom.xml</resource>
-                    <!-- netty's shading of jctools does not remove its pom files -->
-                    <resource>META-INF/maven/org.jctools/jctools-core/pom.properties</resource>
-                    <resource>META-INF/maven/org.jctools/jctools-core/pom.xml</resource>
-                  </resources>
-                </transformer>
-                <!-- Pick up the alternate manifest that was extracted from the core module -->
-                <transformer implementation="org.apache.maven.plugins.shade.resource.IncludeResourceTransformer">
-                  <resource>META-INF/MANIFEST.MF</resource>
-                  <file>${project.build.directory}/META-INF-shaded/MANIFEST.MF</file>
-                </transformer>
-              </transformers>
-              <!-- Keep the dependencies of driver-core -->
-              <promoteTransitiveDependencies>true</promoteTransitiveDependencies>
+              <instructions>
+                <Bundle-SymbolicName>com.datastax.oss.driver.core</Bundle-SymbolicName>
+                <DynamicImport-Package>*</DynamicImport-Package>
+                <!--
+                Don't include because they aren't OSGi bundles, and the driver bundle can
+                live without them:
+                 - jnr
+                 - net.jcip.annotations
+                 - edu.umd.cs.findbugs.annotations
+                Don't include because they are imported by shaded Netty classes
+                but not used by the driver bundle:
+                 - com.google.protobuf
```

Comment:
Because Netty has been shaded, BND now detects a few package imports of Netty dependencies that would otherwise have been treated inside Netty bundles, not here. These are mostly compression libraries. None of them is used by the driver so I just removed them.